ipu-plugin/ipuplugin/cmd/rootcmd.go

```diff
@@ -32,9 +32,9 @@ import (
 const (
 	configFilePath      = "/etc/ipu/"
 	cliName             = "ipuplugin"
-	defaultServingHost  = "localhost"
+	defaultServingAddr  = "/var/run/vendor-plugin.sock"
```

Comment:
nit:
```suggestion
	defaultServingAddr  = "/var/run/ipuplugin.sock"
```
to be consistant - either we change to vendor-plugin or keep ipuplugin throughout, wdyt?

Reply:
I didn't pick the socket name. I used the same name as DPU daemon defined in [here](https://github.com/openshift/dpu-operator/blob/8dd172060f0b7d9cd4ef82b57f583d2dda56f54d/daemon/vendorplugin.go#L16)